lib/rules/custom-property-no-missing-var-function/index.mjs

```diff
@@ -67,19 +67,21 @@ const rule = (primary) => {
 			if (IGNORED_PROPERTIES.has(prop.toLowerCase())) return;
 
 			valueParser(value).nodes.forEach((childNode) => {
-				check(childNode, decl);
+				check(childNode, decl, false);
 			});
 		});
 
 		/**
 		 * @param {Node} node
 		 * @param {import('postcss').Declaration} decl
+		 * @param {boolean} insideStyleQuery - Whether we're inside a style() query
 		 */
-		function check(node, decl) {
+		function check(node, decl, insideStyleQuery) {
```

Comment:
question

It seems `check` is never called with `insideStyleQuery === true`.
Hence `insideStyleQuery` is never set to `true`.

```suggestion
		function check(node, decl, insideStyleQuery = false) {
```
then revert l70 and l105.

What are you planning to use `passStyleQuery`/`insideStyleQuery` for?